packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -151,10 +151,13 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
     return token
   }
 
-  const generateGooglePayToken = async (paymentRequest: google.payments.api.PaymentDataRequest) => {
+  const generateGooglePayToken = async (
+    paymentRequest: google.payments.api.PaymentDataRequest
+  ): Promise<string> => {
+    const environment = window.location.host === 'login.blockchain.com' ? 'PRODUCTION' : 'TEST'
```

Comment:
please get the env from wallet-options selector

Reply:
This is because we actually can only show this one if the host is that one, that I've validated on google.

It can't be used on pre-prod.

Should I still use the selector?